lpa/lpa.h

```diff
@@ -18,9 +18,8 @@ namespace LPA {
         }
         bool isStillAlive();
         bool matchesNameTemplate(QString name);
-        // Note: It's best if the Lea implementation *doesn't* open with full permissions unless necessary, but the Windows impl. was ported.
-        // Still: Only do this if you're sure this is the process you want. Don't want to trigger anticheats.
-        bool canBeginMemoryAccess();
+        // WARNING (for Windows): If this method returns false, this object is no longer valid!
+        bool tryBeginMemoryAccess();
```

Comment:
I'll commit a change which makes the process object invalid on lpa\_unix too, to make the 'for Windows' bit unnecessary...
Oops on my part.